swan-lake/learn-the-language/distinctive-language-features/advanced-general-purpose-language-features.md

```diff
@@ -628,7 +628,7 @@ public function open(string path)
 
 In the above code example, the **``open()``** function has an external implementation since its body is defined as **``= external``**. The ``external`` keyword can be annotated to add additional information that specifies where the implementation comes from.
 
-As part of interfacing with external implementation, Ballerina supports another basic type called ``handle``. A ``handle`` value is a reference to storage managed externally and it may be passed a reference to an external function, like in the JVM where you may have a Java object reference pointing to the implementation.
+As part of interfacing with external implementation, Ballerina supports another basic type called ``handle``. A ``handle`` value is a reference to storage managed externally, and it may be passed as a reference to an external function, like in the **JVM** where you may have a Java object reference pointing to the implementation.
```

Comment:
```suggestion
As part of interfacing with an external implementation, Ballerina supports another basic type called ``handle``. A ``handle`` value is a reference to storage managed externally and it may be passed as a reference to an external function like in the **JVM** where you may have a Java object reference pointing to the implementation.
```